doc/changelog.rst

```diff
@@ -33,6 +34,7 @@ Localization
 Admin
 ~~~~~
 
+- Enable all methods for order creator
```

Comment:
Actually it's still not good description of the change, since not *all* are enabled, just those that have status=enabled. How about "Make all enabled shipping and payment methods available in order creator" then?